document/js-api/index.bs

```diff
@@ -687,6 +695,62 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
     1. Return undefined.
 </div>
 
+<h3 id="globals">Globals</h3>
+
+<pre class="idl">
+dictionary GlobalDescriptor {
+  required USVString type;
+  required Number value;
+  boolean mutable;
+};
+
+[LegacyNamespace=WebAssembly, Constructor(GlobalDescriptor descriptor), Exposed=(Window,Worker,Worklet)]
+interface Global {
+  // TODO: How to define @@toPrimitive hook?
+  // TODO: Types for value getter/setter? (any? union?)
```

Comment:
What are the getter/setters supposed to be?

Reply:
The getter for 'value' reads the global value; the setter sets the value iff the global is mutable.